docs/api/buckets.rst

```diff
@@ -0,0 +1,199 @@
+Working with buckets
+====================
+
+:ref:`Buckets <buckets>` enable the creation of collections handled by
+a group of users.
+
+As soon as there is a need to let people work on the same records set
+or to work with the same collection, the solution is to create a :ref:`bucket <buckets>`.
+
+:ref:`Buckets <buckets>` gives a public unique name to the collection.
+
+The data is not anymore linked to a specific user that only has access
+to her private data but is linked to the bucket and managed by
+bucket's owners (those who have the ``write_bucket`` permission on the
+bucket.
+
+Access to buckets, groups, collections and records are granted using
+:ref:`permissions <permissions>`.
+
+
+Creating a new bucket
+---------------------
+
+``/buckets/<bucket_id>``
+
+This endpoint defines the bucket resource.
+
+* Handle the owner list as well as the buckets permissions
+
+
+POST /buckets
+'''''''''''''
+
+**Requires authentication**
+
+This endpoint creates a new bucket with a generated unique id.
+
+By default the user id is used as the only owner.
+
+**Optional parameters**
+
+- ``permissions``: A mapping object that defines the list of user for
+  each permission
+ - ``write_bucket``: The list of users principals that have
+   administration permissions on the bucket, the creator is
+   automatically added to the owner list.
+ - ``create_groups``: Permission to create new groups
+ - ``create_collections``: Permission to create new collections
```

Comment:
We probably want to have all the combination of object / permission, and probably remove the ones which don't make sense. This would be more future proof than having write / read and then not being able to be more fine grained than that.